assets/sass/style.scss

```diff
@@ -61,6 +61,7 @@
 @import "04-elements/forms";
 @import "04-elements/media";
 @import "04-elements/misc";
+@import "04-elements/links";
```

Comment:
Please also include the new file in the table of contents :)

Reply:
Which table of contents are you referring to? 

Looks like links are already listed in the TOC of this file: https://github.com/WordPress/twentytwentyone/blob/76ca10c37ba9be894be64467b2d9d854d6c8d235/assets/sass/style.scss#L22

Reply:
Yeah, the toc and the actual files in the folder are out of sync